ts/components/messages/MessageDetail.tsx

```diff
@@ -51,15 +55,43 @@ const styles = StyleSheet.create({
   webview: {
     marginLeft: variables.contentPadding,
     marginRight: variables.contentPadding
+  },
+  messageIDContainerText: {
+    fontSize: variables.fontSizeSmaller,
+    display: "flex",
+    alignItems: "center"
+  },
+  messageIDLabelContainer: {
+    display: "flex",
+    alignItems: "center",
+    flexDirection: "row"
+  },
+  messageIDLabelText: {
+    fontSize: variables.fontSizeSmall
```

Comment:
we should probably add the style that keeps the text on one line

Reply:
You're right, I've attached a screenshot of iPhone 5s' screen and I see the point. Considering that by using this layout there would be no enough space to keep ID on a single line, how do you suggest to take action? @matteodesanti 

![screen](https://user-images.githubusercontent.com/39746618/56643865-d2c9e200-667a-11e9-9095-6c3c6a439566.png)

Reply:
Like here https://github.com/teamdigitale/io-app/pull/981/files#diff-2b0d546ebe434a9b4fab24bedd68c1aeR170

Reply:
Considering that ellipsis would be used for presentation only, I've put text on one line. "Copy" button still copies the whole id.

![screen2](https://user-images.githubusercontent.com/39746618/56644381-dad65180-667b-11e9-9ad6-6632d88cb417.png)

Reply:
the idea was also to have the full ID visible in case the user takes a screenshot with Instabug (see my other comment about Instabug attributes as a possible fix to this issue)

Reply:
As @matteodesanti suggested, string MUST NOT be truncated.